CODE/explain_human_resps.py

```diff
@@ -76,28 +199,12 @@ def main(argv):
         tf_feat_mtx_dict[feat_info_dict['tfs'][0]].shape,
         nontf_feat_mtx.shape))
 
-    ## Model prediction and explanation
-    tfpr_explainer = TFPRExplainer(
-        tf_feat_mtx_dict, nontf_feat_mtx, features, 
-        label_df_dict, filepath_dict['output_dir'], model_hyparams
-    )
-    
-    if args.model_tuning:
-        logger.info('==> Tuning model hyperparameters <==')
-        tfpr_explainer.tune_hyparams()
-
-    logger.info('==> Cross validating response prediction model <==')
-    tfpr_explainer.cross_validate()
+    run_tfpr(tf_feat_mtx_dict, nontf_feat_mtx, features, label_df_dict, filepath_dict['output_dir'], model_hyparams, args.model_tuning, args.permutations, args.number_of_permutations, args.disable_shap)
 
-    # TODO: to be removed in release
-    if not args.disable_shap:
-        logger.info('==> Analyzing feature contributions <==')
-        tfpr_explainer.explain()
-    
-    logger.info('==> Saving output data <==')
-    tfpr_explainer.save()
-    
-    logger.info('==> Completed <==')
+    #if args.permutations:
+    #    run_permutations(tf_feat_mtx_dict, nontf_feat_mtx, features, label_df_dict, filepath_dict['output_dir'], model_hyparams, args.model_tuning, args.number_of_permutations, args.enable_permutation_shap)
+    #else:
+    #    run_standard(tf_feat_mtx_dict, nontf_feat_mtx, features, label_df_dict, filepath_dict['output_dir'], model_hyparams, args.model_tuning, args.disable_shap)
```

Comment:
Are these two intended to be use?

Reply:
Nope. These have been removed.

Reply:
Should their unused function definitions also be removed?

Reply:
Ah, I just amended those changes as well.